website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -48,15 +48,20 @@ Please read through the entire procedure before beginning the upgrade. There are
 
```

Comment:
Let's add more info on the 3. step as an overview for what's happening. 

3. Set environment variables for your cluster to upgrade to the latest patch version of the current Karpenter version you're running on.